unpackaged/config/qa/applications/___NAMESPACE___HEDA.app

```diff
@@ -14,9 +14,4 @@
     <tabs>%%%NAMESPACE%%%Program_Plan__c</tabs>
     <tabs>%%%NAMESPACE%%%Trigger_Handler__c</tabs>
     <tabs>%%%NAMESPACE%%%HEDA_Settings</tabs>
-    <tabs>%%%NAMESPACE%%%Error__c</tabs>
```

Comment:
Are these intentionally being removed? I thought Dina originally added these to the app for easy access, just thought I'd ask here.